solr/solrj/src/java/org/apache/solr/common/params/CommonAdminParams.java

```diff
@@ -27,6 +27,8 @@
   String IN_PLACE_MOVE = "inPlaceMove";
   /** Method to use for shard splitting. */
   String SPLIT_METHOD = "splitMethod";
+  /** Key to use during shard splitting */
+  String SPLIT_KEY = "split.key";
```

Comment:
Should this be `splitKey` in a perfect world?

Reply:
That'd be my preference, yep.  This constant only uses dot-delimiting since that's what the v1 API used up to this point.  And it's only used for the v1 API, so there's the convention doesn't spill into v2-land.